doc/design/OrionMySQLSink.md

```diff
@@ -1,60 +1,67 @@
+According to the [naming conventions](naming_conventions.md), a database named as the `fiware-service` header value within the event is created (if not existing yet).
 
-For instance, if the chosen table type is by destination and the grouping feature is not enabled then two different entities data, `car1` and `car2` both of type `car` will be persisted in two different MySQL tables, according to their <i>default</i> destination, i.e. `car1_car` and `car2_car`, respectively. However, if a grouping rule saying "all cars of type `car` will have a modified destination named `cars`" is enabled then both entities data will be persisted in a single table named `cars`. In this example, the direct aggregation is determined by the table type (by destination), but inderectly we have been deciding the aggregation as well through a grouping rule.
+Then, the context responses/entities within the container are iterated, and a table is created (if not yet existing) within the above database whose name depends on the configured table type:
 
-[Top](#top)
+* `table-by-destination`. A table named as the concatenation of `<fiware_servicePath>_<destination>` is created (if not yet existing).
+* `table-by-service-path`. A table named as the `<fiware-servicePath>` is created (if not yet existing). 
 
-####Important notes regarding the persistence mode
-Please observe not always the same number of attributes is notified; this depends on the subscription made to the NGSI-like sender. This is not a problem for the `row` persistence mode, since fixed 8-fields data rows are inserted for each notified attribute. Nevertheless, the `column` mode may be affected by several data rows of different lengths (in term of fields). Thus, the `column` mode is only recommended if your subscription is designed for always sending the same attributes, event if they were not updated since the last notification.
+The context attributes within each context response/entity are iterated, and a new data row (or rows) is inserted in the current table. The format for this row depends on the configured persistence mode:
 
-In addition, when running in `column` mode, due to the number of notified attributes (and therefore the number of fields to be written within the Datastore) is unknown by Cygnus, the table can not be automatically created, and must be provisioned previously to the Cygnus execution. That's not the case of the `row` mode since the number of fields to be written is always constant, independently of the number of notified attributes.
+* `row`: A data row is added for each notified context attribute. This kind of row will always contain 8 fields:
+    * `recvTimeTs`: UTC timestamp expressed in miliseconds.
+    * `recvTime`: UTC timestamp in human-redable format ([ISO 6801](http://en.wikipedia.org/wiki/ISO_8601)).
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/9a1b0b52da721a98c8997d944608e2b01bf85eeb